spec/beaker-hostgenerator/generator_spec.rb

```diff
@@ -98,12 +57,20 @@ module BeakerHostGenerator
             fixture_hash['environment_variables'].each do |key, value|
               ENV[key] = value
             end
+
             cli = BeakerHostGenerator::CLI.new(arguments)
-            test_hash = YAML.load(cli.execute)
+            if fixture_hash['expected_exception']
+              # Turn fully-qualified classname string into an actual Class object.
+              expected_class = eval fixture_hash['expected_exception']
```

Comment:
Updated this here: https://github.com/puppetlabs/beaker-hostgenerator/pull/28/commits/b0926b2216f419bbd56ddbed71dc418efeaa822f#diff-8bdb9a6d73949911cc906d87d7f4aef8R67